subsys/net/lib/nrf_cloud/include/nrf_cloud_agps.h

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I see that this header is used in samples, shouldn't it be moved into `include/net` then? That's a place for public headers.

Reply:
Agreed, moved now